src/layouts/Home/components/Search.tsx

```diff
@@ -78,7 +79,7 @@ export const Search: React.FC = () => {
                 <h1
                   css={css`
                     margin-bottom: ${p => p.theme.space.md};
-                    line-height: ${p => p.theme.space.base * 12.5}px;
+                    line-height: ${p => getLineHeight('56px', `${p.theme.space.base * 12}px`)};
```

Comment:
The `fontSize` variable should be extracted out and shared between this and `font-size`. The 56px height should be expressed as a `space.base` calculation.

Reply:
I wasn't sure how to separate this out within a `css` prop, but figured it out. Updated.